qaframework-bdd-tests/src/test/resources/features/refapp-2.x/02-login/verifyLogin.feature

```diff
@@ -0,0 +1,10 @@
+Feature: User based login
+
```

Comment:
would be a good implementation if we have a background of the scenario.

Reply:
For some reasons. We dont need a back ground scenario here. One reason is because method called in before method does it for all the initiating the page. in this case what would go in background case would ideally be intiating home page and stuff which is by default done by initiateLogin. thats the reason. Let me know if you need more clarification.